suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -668,3 +671,18 @@ export const selectIsSelectedDeviceImported = (state: DeviceRootState) => {
     const device = selectDevice(state);
     return device?.id === PORTFOLIO_TRACKER_DEVICE_ID;
 };
+
+export const selectDeviceLabel = (state: DeviceRootState, id: TrezorDevice['id']) => {
+    const device = selectDeviceById(state, id);
+    return device?.label ?? null;
+};
+
+export const selectDeviceName = (state: DeviceRootState, id: TrezorDevice['id']): string | null => {
+    const device = selectDeviceById(state, id);
+    return device?.name ?? null;
+};
+
+export const selectSelectedDevicesName = (state: DeviceRootState) => {
```

Comment:
It's just plural, like selectMartinsName - 'vyber martinovo jmeno'

If it was multiple names, than `selectDevicesNames`?

Reply:
Oh, I didnt read apostrophe there. I would prefer without `s` to be more obvious. but up to you here

Reply:
actually, it makes sense to remove the s to be consistent. otherwise we should rename selectDeviceName to selectDevicesName according to your point. So I suggest to remove the s to a) follow the current naming approach b) make it clear on first sight

Reply:
Hear your point. I think it's better to have the `s` there but without the apostrophe, I guess it's not as clear. Changed here 👌
https://github.com/trezor/trezor-suite/pull/9749/commits/c693b8e08ea8ff68d8953d053f72c3db22e55ba1